library/translation.inc.php

```diff
@@ -321,4 +321,21 @@ function mb_strpad($input, $length, $pad = ' ', $type = STR_PAD_RIGHT, $charset
 return $output;
 }
 
+
+/**
+ * Set theme for RTL language direction in globals setting.
+ * @return {void}
+ * @author Amiel <amielel@matrix.co.il>
+ */
+function setRtlCssTheme(){
+    
+    //validate that user id exist 
+    if(!isset($_SESSION['authUserID']))return;
+     
+    if(!function_exists('setUserSetting'))require_once dirname(__FILE__) . '/user.inc';
+    
+    //set rtl theme 
+    setUserSetting('global:css_header', 'style_sky_blue_rtl.css', null, false); 
+}
```

Comment:
A follow up question to my above comment. Does one need to keep LTR and RTL themes separately or can they be combined in the same theme css file? Just trying to figure this out to sort the best mechanism for selecting the theme.